membership/src/test/scala/zio/membership/transport/TcpTransportSpec.scala

```diff
@@ -0,0 +1,48 @@
+package zio.membership.transport
+
+import zio._
+import zio.test._
+import zio.test.Assertion._
+import zio.clock.Clock
+import zio.duration._
+import zio.macros.delegate._
+import zio.nio.SocketAddress
+import zio.test.environment.TestClock
+import zio.test.environment.Live
+import zio.test.TestAspect._
+
+object TransportSpec
+    extends DefaultRunnableSpec({
```

Comment:
I'm looking forward to having inheritance-based specs here.

Reply:
Yeah, they are much less awkward